### PR TITLE
fix spec syntax

### PR DESCRIPTION
**Pull request by omertoast**
I came across a syntax error from this line while using [openapi-typescript](https://github.com/drwpow/openapi-typescript)

### CODE REVIEW

**Comment by wilsonianb** [comment]
Previous discussion: https://github.com/interledger/open-payments/pull/168#discussion_r961773935

File: auth-server-open-api-spec.yaml [hunk 1]
```diff
@@ -577,7 +577,9 @@ components:
           description: '[ISO8601 repeating interval](https://en.wikipedia.org/wiki/ISO_8601#Repeating_intervals)'
       required:
         - interval
-        - oneOf - sendAmount - receiveAmount
+        - oneOf
+        - sendAmount
+        - receiveAmount
```
Comment (wilsonianb) [inline auth-server-open-api-spec.yaml:582]: What do we think of:
```
anyOf:
  - required:
    - interval
    - sendAmount
  - required:
    - interval
    - receiveAmount
```
https://stackoverflow.com/questions/41172240/json-schema-how-to-make-anyof-two-or-more-properties-required

Comment (wilsonianb) [reply auth-server-open-api-spec.yaml:582]: :point_up_2: that didn't work.

:point_down: this should:
- https://github.com/interledger/open-payments/pull/180

**Comment by wilsonianb** [comment]
> I came across a syntax error from this line while using [openapi-typescript](https://github.com/drwpow/openapi-typescript)

How are you planning to use the generated types?
I had previously (unsuccessfully) tried to generate types using [oats](https://github.com/smartlyio/oats), but it looks like `openapi-typescript` works with our `open-api-spec.yaml`. :tada: 
I might look into replacing our manually defined types in Rafiki (example: https://github.com/interledger/rafiki/blob/e96d44e692660babd4e6c45d6e36178e9ee01f86/packages/backend/src/open_payments/payment/incoming/model.ts#L248-L260)

File: auth-server-open-api-spec.yaml (hunk 1 above)
Comment (omertoast) [reply auth-server-open-api-spec.yaml:582]: I don't know the correct spec, I was just trying to get syntax work so yours is probably the right one.

**Comment by omertoast** [comment]
> > I came across a syntax error from this line while using [openapi-typescript](https://github.com/drwpow/openapi-typescript)
> 
> How are you planning to use the generated types? I had previously (unsuccessfully) tried to generate types using [oats](https://github.com/smartlyio/oats), but it looks like `openapi-typescript` works with our `open-api-spec.yaml`. 🎉 I might look into replacing our manually defined types in Rafiki (example: https://github.com/interledger/rafiki/blob/e96d44e692660babd4e6c45d6e36178e9ee01f86/packages/backend/src/open_payments/payment/incoming/model.ts#L248-L260)

I'm creating a request validator service for the [rafiki-auth](https://github.com/omertoast/rafiki-auth) package @adrianhopebailie and I work on. I can do static type checking using [openapi-typescript](https://github.com/drwpow/openapi-typescript) beside runtime type checking. So we don't have to create validate functions to get static type checks like to auth package in Rafiki.

**Pull request closed without merging**